packages/ibm-products/src/components/BigNumbers/BigNumbers.stories.jsx

```diff
@@ -157,20 +173,3 @@ export const bigNumbers = Template.bind({});
 bigNumbers.args = {
   ...defaultProps,
 };
-
-export const withEditButton = Template.bind({});
```

Comment:
With the added ability to show/hide the Edit button, this template is no longer req'd.